scripts/disasm

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# disasm <amacc ELF executable file>
+dd bs=1 if=$1 of=$1.asmtmp `readelf -a $1 2>/dev/null | awk '/\.text.*PROGBITS/ { s = sprintf("skip=%d", "0x" $6) ; c = sprintf("count=%d", "0x" $7) ; print s, c }'`
```

Comment:
Can you split this line into multiple lines? Thus, we can review and maintain in easier way.

Reply:
I might be able to, but my main worry is that it will be more error prone to do that if someone tries to modify the script for their own purposes.  There is a lot of punctuation  and pipe magic going on there that is very fragile.  Also, for all the splits I try, it actually becomes less readable.  Any specific suggestions?

Reply:
At least, you can move the results of `readelf` and `awk` to `$DD_OPTS`, which is expectedly prior to the line starting with `dd` command.

Reply:
Yoe know, there is no reason to use hex numbers for the skip and count arguments since they are hidden.  That would shrink that line.

Reply:
sorry, I have to take that back.  It either needs what I used, or a hex to decimal conversion.